bee-node/config.example.toml

```diff
@@ -99,7 +98,7 @@ delay   = 60480
 address = "tcp://localhost:1883"
 
 [dashboard]
-port  = 8081
+bind_addresses  = "/ip4/0.0.0.0/tcp/8081"
```

Comment:
As per the other discussion, this should either be singular or an array. I don't think there is any usecase where we would need to bind many addresses for the dashboard so I would go the singular way.

Reply:
Agree fully, singular or array seems nice.